src/Kdyby/Annotations/DI/AnnotationsExtension.php

```diff
@@ -81,7 +82,7 @@ public function getConfig(array $defaults = NULL, $expand = TRUE)
 			$config = Nette\DI\Config\Helpers::merge($config, ['ignore' => $globalConfig['doctrine']['ignoredAnnotations']]);
 		}
 
-		return $this->compiler->getContainerBuilder()->expand($config);
+		return Nette\DI\Helpers::expand($config, $this->getContainerBuilder()->parameters);
```

Comment:
Thanks, that's it! Interesting, I will try using this.

Do you use this anywhere in your extensions?